sql/item_armor.sql

```diff
@@ -9879,26 +9879,26 @@ INSERT INTO `item_armor` VALUES (20763,'kbiroj',99,119,2097345,65,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20764,'crobaci_+2',99,119,2097344,0,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20765,'tunglmyrkvi',99,119,2097345,377,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20766,'eminent_sword',99,117,2097345,0,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20767,'medicor_sword',99,109,2097345,0,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20767,'medicor_sword',99,109,2097345,64,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20768,'kaquljaan',99,115,2097345,0,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20769,'xiphias_-1',99,0,192,0,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20770,'mesyohi_sword',99,119,2097345,0,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20771,'cama._claymore',99,109,2097344,0,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20772,'voay_sword_-1',99,105,2097344,0,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20773,'voay_sword',99,107,2097344,0,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20774,'voay_sword_+1',99,108,2097344,0,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20772,'voay_sword_-1',99,105,2097344,626,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20773,'voay_sword',99,107,2097344,626,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20774,'voay_sword_+1',99,108,2097344,626,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20775,'crobaci_+1',99,113,2097344,432,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20776,'beorc_sword',99,109,2097152,0,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20777,'f.front_claymore',99,106,2097344,0,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20778,'bereaver',99,113,2097344,0,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20779,'senbaak_nagan',99,117,2097344,0,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20780,'senbaak_nagan_+1',99,118,2097344,0,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20779,'senbaak_nagan',99,117,2097344,73,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20780,'senbaak_nagan_+1',99,118,2097344,73,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20781,'sowilo_claymore',1,0,2097152,0,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20782,'coalition_sword',99,100,2097344,629,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20783,'uruz_blade',75,0,2097152,72,0,0,1,0);
 INSERT INTO `item_armor` VALUES (20784,'uruz_blade_+1',75,0,2097152,72,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20785,'thurisaz_blade',99,100,2097152,72,0,0,1,0);
-INSERT INTO `item_armor` VALUES (20786,'thurisaz_blade_+1',99,101,2097152,72,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20785,'thurisaz_blade',99,100,2097152,67,0,0,1,0);
+INSERT INTO `item_armor` VALUES (20786,'thurisaz_blade_+1',99,101,2097152,67,0,0,1,0);
```

Comment:
yes, http://www.ffxiah.com/item/20784/uruz-blade-1 and http://www.ffxiah.com/item/20785/thurisaz-blade dont match but it matches http://www.ffxiah.com/item/16584/mythril-claymore and the mid is 67